mllam_data_prep/ops/cropping.py

```diff
@@ -0,0 +1,201 @@
+import numpy as np
+import spherical_geometry as sg
+import xarray as xr
+from spherical_geometry.polygon import SphericalPolygon
+
+
+def _get_latlon_coords(da: xr.DataArray) -> tuple:
+    """
+    Get the latlon coordinates of a DataArray.
+
+    Parameters
+    ----------
+    da : xarray.DataArray
+        The data.
+
+    Returns
+    -------
+    tuple (xarray.DataArray, xarray.DataArray)
+        The latitude and longitude coordinates.
+    """
+    if "latitude" in da.coords and "longitude" in da.coords:
+        return (da.latitude, da.longitude)
+    elif "lat" in da.coords and "lon" in da.coords:
+        return (da.lat, da.lon)
```

Comment:
Better to return longitude, latitude ordering to be consistent with x, y ordering of rest of codebase.

Reply:
I'm not sure what you mean above, I am just talking about the ordering of return arguments
```suggestion
        return (da.longitude, da.latitude)
    elif "lat" in da.coords and "lon" in da.coords:
        return (da.lon, da.lat)
```
(and swapping that ordering in later assignments that call this function)

Reply:
Ah I see :) I thought you meant the order of the coordinates with the `xr.DataArray`s that are returned. Ok yes, we can change that, I just thought it made sense to have that order since people say "lat/lon-values", but we can make it explicit that the longitude are the first elements of the tuple